index.html

```diff
@@ -25,7 +25,7 @@
     <!-- Twitter -->
     
     <meta property="twitter:card" content="summary_large_image">
-    <meta property="twitter:url" content="https://web.yoshitha.tk">
+    <meta property="twitter:url" content="https://x.com/Yoshitha_R">

```

Comment:
Keep this don't change this bcs this is a meta tag @chow870 
```suggestion
    <meta property="twitter:url" content="https://x.com/web.yoshitha.tk">
```